tests/unit/test_templating.py

```diff
@@ -221,3 +221,8 @@ def test_keyset_functions(self):
         variables = {"input": {"body": body}}
         result = ApiGatewayVtlTemplate().render_vtl(template, variables)
         assert result == " b"
+
+    def test_dash_in_variable_name(self):
```

Comment:
Thanks for adding the modified upstream test here.

In addition, can we also add a small test that tests the actual functionality added (variable names with dashes).

Something like this seems to work: 
```
template = "$my-var-1"
result = ApiGatewayVtlTemplate().render_vtl(template, {"my-var-1": 123})
assert result == "123"
```

Variable assignment doesn't seem to work yet (which is probably fine, if we're aware of this limitation):
```
template = "#set($my-var-1 = 123) $my-var-1"
result = ApiGatewayVtlTemplate().render_vtl(template, {})
assert result == " 123"
```
(yields `airspeed.TemplateSyntaxError`)

Reply:
Yep 👍 - variable assignment is a different use case, valid, but another.
Regarding the "actual" functionality test, I added downstream, I can added it here but I didn't forget 😄